src/main/kotlin/io/github/devcrocod/korro/KorroTask.kt

```diff
@@ -1,65 +1,124 @@
 package io.github.devcrocod.korro
 
 import org.gradle.api.DefaultTask
-import org.gradle.api.GradleException
+import org.gradle.api.NamedDomainObjectContainer
+import org.gradle.api.Project
+import org.gradle.api.artifacts.Configuration
 import org.gradle.api.file.FileCollection
-import org.gradle.api.tasks.Delete
-import org.gradle.api.tasks.InputFiles
-import org.gradle.api.tasks.TaskAction
+import org.gradle.api.tasks.*
+import org.gradle.workers.WorkAction
+import org.gradle.workers.WorkerExecutor
+import javax.inject.Inject
 
-abstract class KorroTask : DefaultTask() {
-    private val ext: KorroExtension = project.extensions.getByType(KorroExtension::class.java)
+// TODO get from central version
+const val dokkaVersion = "1.8.10"
```

Comment:
Is it possible to take the versions from the properties? Otherwise, it seems that at some point the versions will diverge